### PR TITLE
Add snippets for Dockerfile, Fish shell, JavaScript, Perl, and Org

### PR DESCRIPTION
**Pull request by sei40kr**
## Dockerfile

- `alpine-dockerize`

## Fish shell

- `bang`
- `block`
- `bp`
- `for`
- `function`
- `if`
- `ife`
- `sw`
- `while`

## JavaScript

- `flow`

## Perl

- `bang`
- `enc`
- Remove the shebang from `use` and rename it `strict`

## Org

- `src`

### CODE REVIEW

File: snippets/perl-mode/strict [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- mode: snippet -*-
+# name: use strict; use warnings
+# key: strict
```
Comment (AndreaCrotti) [inline snippets/perl-mode/strict]: there is already a snippet like that called `use` at the moment.
We could merge them or maybe rename the other one?

File: snippets/org-mode/src [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# -*- mode: snippet -*-
+# name: src
+# key: <src
+# --
+#+BEGIN_SRC $1
```
Comment (AndreaCrotti) [inline snippets/org-mode/src]: This was also added in another PR, I think I'll add it separately

File: snippets/dockerfile-mode/dockerize [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# -*- mode: snippet -*-
+# name: dockerize
+# key: dockerize
+# --
+ENV DOCKERIZE_VERSION v0.6.1
+
+RUN apk update && \
```
Comment (AndreaCrotti) [inline snippets/dockerfile-mode/dockerize]: Didn't know about this dockerize,but it looks like it supports many OS, this is just the Alpine specific version.
At least I would make the name and key more specific.

**Comment by AndreaCrotti** [comment]
Thanks a lot @sei40kr I left a few comments!

**Comment by AndreaCrotti** [comment]
@sei40kr if you could address the few comments I left I'd be happy to merge this PR

**Comment by sei40kr** [comment]
@AndreaCrotti I updated the PR. Please review :pray:

**Comment by AndreaCrotti** [comment]
I'm still not convinced about the `dockerize` snippet, at least it should be clear that it's just the alpine version, can we rename it to reflect that @sei40kr ?

**Comment by sei40kr** [comment]
@AndreaCrotti 
Screenshot from 2019-12-10 23-27-24
I used `yas-choose-text` so that we can select Linux or Alpine Linux when expanding the snippet.

**Comment by AndreaCrotti** [comment]
I think that's not complete because the `linux` (which is ubuntu) doesn't use apk but uses apt.

I kind of prefer if they were two different snippets, or just leave one snippet for alpine for now.

**Comment by sei40kr** [comment]
@AndreaCrotti 
Sorry I've just updated the PR. I renamed it `alpine-dockerize`.

**Comment by AndreaCrotti** [comment]
Nice thanks!

**Pull request merged**